resources/conf/log_config.toml

```diff
@@ -1,16 +1,17 @@
-# The logging configuration file for control plane
-######### root Level ########
-logfile = "logs/adapter.log"
+# The logging configuration for Adapter
+
+## Adapter root Level configurations
+
+logfile = "logs/adapter.log" # This file will be created inside adapter container.
 logLevel = "INFO"
 
 [rotation]
-MaxSize = 10    #meabytes
+MaxSize = 10    # In MegaBytes (MB)
 MaxBackups = 3
-MaxAge =  2   #days
+MaxAge =  2   # In days
 Compress = true
 
-######### package Level ############
-# LogLevels = "DEBG", "FATL", "ERRO", "WARN", "INFO", "PANC"
```

Comment:
Shall we keep this as a reference. As some of these levels defers from log4j log level keys and users might get confused.

Reply:
Fixed https://github.com/wso2/product-microgateway/pull/2006/commits/714c7dbfa8b6ded71ddefc47e7100d9bf9028570